paddle/fluid/distributed/service/communicator.cc

```diff
@@ -64,7 +64,7 @@ void Communicator::init_gflag(const std::string &gflags) {
   flags.insert(it, "exe default");
   char *flags_ptr[flags.size()];
   for (size_t i = 0; i < flags.size(); ++i) {
-    flags_ptr[i] = (char *)(flags[i].c_str());
+    flags_ptr[i] = (char *)(flags[i].c_str());  // NOLINT
```

Comment:
Why these `nolint`s are added?

Reply:
Trigger cpplint check failed because of C-style type cast.

Reply:
> Why these `nolint`s are added?
Trigger cpplint check failed because of C-style type cast.